seatunnel-connectors-v2/connector-flink-table-store/src/main/java/org/apache/seatunnel/connectors/seatunnel/fts/config/FlinkTableStoreConfig.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.connectors.seatunnel.fts.config;
+
+import org.apache.seatunnel.api.configuration.Option;
+import org.apache.seatunnel.api.configuration.Options;
+
+import java.util.List;
+
+public class FlinkTableStoreConfig {
+
+    public static final Option<String> TABLE_PATH =
```

Comment:
IMO, `table_path` config option isn't user-friendly and we could provide warehouse path, database and table.

Reply:
Agree with you, in newest version I changed it to `warehouse` and get table use `Catalog` interface.